### ISSUE TITLE
Foot spasticity 

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#9748]
**Preferred term label:**
Foot spasticity

**Synonyms**
Spastic foot|Spasticity in the foot

**Definition (free text, please give PubMed ID)**
Spasticity (velocity-dependent increase in tonic stretch reflexes with increased muscle tone and exaggerated/hyperexcitable tendon reflexes) in the foot muscles (3216972| 33036356).

**Parent term (use hpo.jax.org/app)**
Lower limb spasticity (HP:0002061)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

**Comment**
The term, Foot spasticity was defined using the definition of Spasticity given in the HPO; PMIDs 3216972 and 33036356 were provided for the usage of the term.

**Comment by pnrobinson** [comment]
To my knowledge there is no really an isolated foot spasticity and this is part of lower extremity spasticity. One can get a foot malformation based on spasticity but this is a different code.
